### PR TITLE
RCAL-129: Document photom & area reference files

### LINKED ISSUE #173: Document photom & area reference files
_Issue [RCAL-129](https://jira.stsci.edu/browse/RCAL-129) was created on JIRA by [David Davis](https://jira.stsci.edu/secure/ViewProfile.jspa?name=ddavis):_

Document the format and parameters for the photom and area reference files.

Initial versions of these files are at ███████████████████████████████████

This should be in <https://roman-cal-pipeline.readthedocs.io/en/latest/roman/package_index.html> and title should be "Photometric Calibration"

### PR DESCRIPTION
**Pull request by PaulHuwe**
GitHub issue, Closes #173 

Resolves [RCAL-129](https://jira.stsci.edu/browse/RCAL-129)

**Description**

This PR addresses adding documentation for PHOTOM and Area reference files, which required placeholder documentation for the photom step. In addition, I fixed an improper object in dark documentation.

Checklist
- [ ] Tests

- [x] Documentation

- [x] Change log

- [x] Milestone

- [x] Label(s)

### CODE REVIEW

File: docs/roman/references_general/area_reffile.inc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Reference File Format
++++++++++++++++++++++
+AREA reference files are ASDF format, with 2D data arrays.
+The format and content of the file is as follows
+(see `~roman_datamodels.datamodels.PixelareaRefModel`):
+
+=======  ============ ==========================  =============
+Data      Array Type   Dimensions                  Data type
+=======  ============ ==========================  =============
+area      NDArray      4096 x 4096                 float32
```
Comment (ddavis-stsci) [inline docs/roman/references_general/area_reffile.inc:39]: I think the data have been trimmed to 4088x4088 by this step.

File: docs/roman/references_general/photom_reffile.inc [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+And the variable attributes for the columns (with data type):
++------------+----------------+-----------+------------+------------------------+
+| Instrument | Column name    | Data type | Dimensions | Units                  |
++============+=======+================+===========+============+================+
+| WFI        | photmjsr       | float     | scalar     | MJy/steradian          |
++            +----------------+-----------+------------+------------------------+
+|            | uncertainty    | float     | scalar     | MJy/steradian          |
++            +----------------+-----------+------------+------------------------+
+|            | pixelareasr    | float     | scalar     | MJy/steradian          |
```
Comment (ddavis-stsci) [inline docs/roman/references_general/photom_reffile.inc:52]: Shouldn't pixel area just be steradian?

**Review by ddavis-stsci — APPROVED** [review]
A few minor suggestions but looks good overall.

File: docs/roman/references_general/photom_reffile.inc (hunk 2 above)
Comment (PaulHuwe) [reply docs/roman/references_general/photom_reffile.inc]: Changed.

**Pull request merged**